README.md

```diff
@@ -56,7 +56,9 @@ For videos, protected from embed by "Specific domains" you can have an issue
 with thumbnails download to drupal media. In this case - apply a patch
 for drupal core:
 
-* _patches/OEmbed\_vimeo\_private\_videos.patch_
+* _patches/OEmbed\_vimeo\_private\_videos.patch_ - in case of using core media
+* _patches/video\_embed\_field\_vimeo\_private\_videos.patch_ - in case of
```

Comment:
considering to add these via composer patches?

Reply:
We can add these to Open Y, but it will be useful only for clients with Pro/Business/Premium Vimeo plans and with enabled domain-level privacy

Reply:
Got it.